spec/book_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../lib/book'
+
+describe Book do
+  context 'we create the book object and test it ' do
+    let(:book) { Book.new('C++', 'Safdari') }
+    it 'Book title must be C++' do
+      expect(book.title).to eq('C++')
+    end
+    it 'Book author must be C++' do
+      expect(book.author).to eq('Safdari')
+    end
+  end
+end
```

Comment:
- Good job done so far, but according to the project requirements, you are required to have unit tests for all the methods in all the classes. Please add tests for the `add_rental` method in your `Book` class 😄